### ISSUE TITLE
Segfault when calling glTexImage2DMultisample using EGL and GLES2

### ISSUE THREAD

**Issue by mrfixit2001** [gonetz/GLideN64#2624]
Everything works fine with multisampling disabled, but with it enabled a segfault occurs.

GDB trace:
https://dpaste.com/6ZZR7ANSL

I can confirm removing this line fixes the segfault, of course then the graphics are screwed up haha
https://github.com/gonetz/GLideN64/blob/master/src/Graphics/OpenGLContext/opengl_TextureManipulationObjectFactory.cpp#L72

Possibly some functions got missed in initGLFunctions for EGL? I tried adding the TexImage2DMultisample call in for EGL but it did not resolve the issue.

**Comment by fzurita** [comment]
If I remember correctly, you need GLES 3.0 minimum for multi sampling.

**Comment by fzurita** [comment]
Yeah, according to this: https://khronos.org/registry/OpenGL-Refpages/es3.1/html/glTexStorage2DMultisample.xhtml

glTexStorage2DMultisample only became part of the base GLES API on GLES 3.1. Older versions may support it through extensions.

**Comment by mrfixit2001** [comment]
I do see the gles2 ext header including these options:
   glTexStorageMem2DMultisampleEXT
   glTextureStorageMem2DMultisampleEXT
Pretty sure those are essentially the same, but their parameters are slightly different than glTexImage2DMultisample - which would make implementing them more complicated. 

More importantly, multisampling gles2 is currently working on both the Gles2n64 and rice cores. So it’s definitely implementable. At a glance I don’t see either core calling any of these functions, just setting the core setting. Hmm
